CHANGELOG.md

```diff
@@ -309,360 +339,7 @@ You can participate in the conversation around this release in [this issue](ADD)
 - Remove cxx from iOS project ([46d6e2a600](https://github.com/facebook/react-native/commit/46d6e2a600f0a4b2a2b55d6f7639a5cfce6a798c) by [@zhongwuzw](https://github.com/zhongwuzw))
 - Restore RCTText tvOS pod compatibility ([5f53cb0b85](https://github.com/facebook/react-native/commit/5f53cb0b8595d5b673dc22f3249960fd012dab18) by [@benoitdion](https://github.com/benoitdion))
 - Fixes animated gifs incorrectly looping ([6f2e6f170e](https://github.com/facebook/react-native/commit/6f2e6f170e3ee785d1ba844971447ea24f91185e) by [@zhongwuzw](https://github.com/zhongwuzw))
-
-### Unknown
-
-- Add Intl to eslint globals. ([f6a62f9ae2](https://github.com/facebook/react-native/commit/f6a62f9ae2278c0f3a1e5c1a6ec3b7cca3421a41))
```

Comment:
One of the other added commits you left in was about a change to eslint, adding a new rule. Are changes like this different?

Reply:
Yeah I thought this was for our internal eslint, not the community lint. Added back.